configurations/local/logistic.py

```diff
@@ -7,12 +7,36 @@
 """
 from configurations.default import *
 import os
-EXPERIMENT='urgcms_excl_hdia_nbinj'
-MODELPATH+=EXPERIMENT+'/'
+
```

Comment:
Algorithm config files should call an experiment config file. Adress that in documentation when solving #13